CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 6.0.0-alpha.1
+
+This release of Teleport introduces Database Access with PostgreSQL support.
```

Comment:
Can you add a block to this about it being a pre-release.

Reply:
Done.